The derive feature can cause duplicate import errors

The [docs](https://docs.rs/zerocopy/latest/zerocopy/#cargo-features) state the following:

> However, you may experience better compile times if you instead directly depend on both zerocopy and zerocopy-derive in your Cargo.toml, since doing so will allow Rust to compile these crates in parallel. To do so, do not enable the derive feature, and list both dependencies in your Cargo.toml with the same leading non-zero version number; e.g:

However, this can lead to issues if another dependency in the chain imports zerocopy *with* the derive feature, as now you suddenly have two macros in scope.

```
error[E0252]: the name `AsBytes` is defined multiple times
 --> /run/build/abc/cargo/vendor/gvdb-0.7.0/src/read/hash.rs:9:23
  |
8 | use zerocopy::{AsBytes, FromBytes};
  |                ------- previous import of the macro `AsBytes` here
9 | use zerocopy_derive::{AsBytes, FromBytes, FromZeroes};
  |                       ^^^^^^^--
  |                       |
  |                       `AsBytes` reimported here
  |                       help: remove unnecessary import
  |
  = note: `AsBytes` must be defined only once in the macro namespace of this module
```

which is then only solveable within the dependency that imports zerocopy_derive in this way.

Therefore this usage should probably be discouraged, or warned against for anything other than binary crates.

--- a/README.md
+++ b/README.md
@@ -114,6 +114,11 @@ for network parsing.
   zerocopy-derive = "0.X"
   ```
 
+  To avoid the risk of [duplicate import errors][duplicate-import-errors] if
+  one of your dependencies enables zerocopy's `derive` feature, import
+  derives as `use zerocopy_derive::*` rather than by name (e.g., `use
+  zerocopy_derive::FromBytes`).
+
 - **`simd`**
   When the `simd` feature is enabled, `FromZeros`, `FromBytes`, and
   `IntoBytes` impls are emitted for all stable SIMD types which exist on the
@@ -127,6 +132,7 @@ for network parsing.
   available on nightly. Since these types are unstable, support for any type
   may be removed at any point in the future.
 
+[duplicate-import-errors]: https://github.com/google/zerocopy/issues/1587
 [simd-layout]: https://rust-lang.github.io/unsafe-code-guidelines/layout/packed-simd-vectors.html
 
 ## Security Ethos

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -114,6 +114,11 @@
 //!   zerocopy-derive = "0.X"
 //!   ```
 //!
+//!   To avoid the risk of [duplicate import errors][duplicate-import-errors] if
+//!   one of your dependencies enables zerocopy's `derive` feature, import
+//!   derives as `use zerocopy_derive::*` rather than by name (e.g., `use
+//!   zerocopy_derive::FromBytes`).
+//!
 //! - **`simd`**
 //!   When the `simd` feature is enabled, `FromZeros`, `FromBytes`, and
 //!   `IntoBytes` impls are emitted for all stable SIMD types which exist on the
@@ -127,6 +132,7 @@
 //!   available on nightly. Since these types are unstable, support for any type
 //!   may be removed at any point in the future.
 //!
+//! [duplicate-import-errors]: https://github.com/google/zerocopy/issues/1587
 //! [simd-layout]: https://rust-lang.github.io/unsafe-code-guidelines/layout/packed-simd-vectors.html
 //!
 //! # Security Ethos

